liquidprompt

```diff
@@ -2341,8 +2347,22 @@ _lp_wifi() {
 # Get the battery status in percent.
 case "$LP_OS" in
     Linux)
-    _lp_battery() {
-        (( LP_ENABLE_BATT )) || return 5
+    __lp_battery_sysfs() {
+        local power_supply
+        for power_supply in "${_LP_LINUX_POWERSUPPLY_PATH}/"*; do
+            if [[ -r "${power_supply}/type" && $(< "${power_supply}/type") == 'Battery' && \
+                -r "${power_supply}/present" && $(< "${power_supply}/present") == '1' && \
+                -r "${power_supply}/status" && -r "${power_supply}/capacity" && \
+                ! -f "${power_supply}/scope" ]]; then
```

Comment:
Add a comment in here about why `/scope` should not exist. Both for me, because I can't find any documentation on it, but also for future reference.

Reply:
Will do, I will give a real world example, from my system.